src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -963,11 +963,12 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
             }
             
             // If question specific sorting is not needed, responses are sorted
-            // by default order (first by team name, then by display name)
+            // in the following order (first by recipient name, then by team name,
+            // then by display name)
             if (questionDetails.isQuestionSpecificSortingRequired()) {
                 Collections.sort(responseRows, questionDetails.getResponseRowsSortOrder());
             } else {
-                responseRows = InstructorFeedbackResultsResponseRow.sortListWithDefaultOrder(responseRows);
+                responseRows = InstructorFeedbackResultsResponseRow.sortListByRecipientNameThenDefaultOrder(responseRows);
```

Comment:
@swxsw, not very sure what the question mark and circles mean in the screenshot. Sorry if my screenshot and explanation wasn't clear! Only the "Response Summary" table (which contains the statistics) should be modified in this issue. 

There are 2 types of constant sum question, one where the giver distributes points among options (such as in my screenshot), and one where the giver distributes points among the recipients of the question (e.g. to team members). The issue title's "recipient name or option" actually just means the options, I guess this caused a lot of confusion.